README.md

```diff
@@ -45,7 +45,12 @@ String MainThermostatHeatingCoolingMode "Main Thermostat Heating/Cooling Mode" (
 
 If items are modified in openHAB, a refresh in Mycroft is needed by the command:
 
-- *"Hey Mycroft, refresh openhab items"*
+- *"Hey Mycroft, refresh openhab items"* or
+- *"Hey Mycroft, refresh open hab items"*
+
+If you've forgotten what items have been identified, you can ask Mycroft:
+- *"Hey Mycroft, list openhab items"* or
+- *"Hey Mycroft, list open hab itmes"*

```

Comment:
It should only be "openHAB". How the person commanding mycroft pronounces openHAB should be irrelevant here.

Reply:
I completely agree, but when I first tried using this skill, Mycroft would never understand me as it would transcribe my "openhab" as "openhab" and it understood only "open hab". So, I was just following what was originally there. If you still think that I shouldn't do that, I'll remove the `open hab` from both readme and implementation.

Reply:
I agree with @ThomDietrich . It's a "code trick" to make mycroft understand openHAB word in any case, but the documentation shouldn't mention that.